lib/puppet/provider/package/yum.rb

```diff
@@ -251,6 +251,7 @@ def latest
       return "#{upd[:epoch]}:#{upd[:version]}-#{upd[:release]}"
     else
       # Yum didn't find updates, pretend the current version is the latest
```

Comment:
`s/pretend/assume/` ?

Reply:
Should I change 5 years old comment line for that?

Reply:
Ah sorry, didn't notice it was here before. You can ignore me.